lib/src/node/legacy.dart

```diff
@@ -357,7 +357,7 @@ RenderResult _newRenderResult(
   var end = DateTime.now();
 
   var css = result.css;
-  Uint8List? sourceMapBytes;
+  Uint8List? sourceMapBytes = undefined;
```

Comment:
I think this needs to be `dynamic` to satisfy the type checker since `undefined` isn't null.

Reply:
Oh, right, I guess https://github.com/pulyaevskiy/node-interop/pull/109 hasn't been released yet.